12/umbraco-cms/tutorials/creating-a-property-editor/part-2.md

```diff
@@ -79,7 +79,7 @@ Your `package.manifest` file should now look something like this:
         "hideLabel": true,
         /*Optional: Add 'read-only' support. Available from Umbraco 10.2+*/
         "supportsReadOnly": true
-      }, // Remeber a comma seperator here at the end of the editor block!
+      }, // Remember a comma separator here at the end of the editor block!
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?